package.json

```diff
@@ -38,6 +37,7 @@
 		"@nestjs/core": "^7.4.4",
 		"@nestjs/platform-express": "^7.4.4",
 		"@nestjs/testing": "^7.4.4",
+		"@types/amqplib": "^0.5.15",
```

Comment:
I expressly added types to dependencies. This way all amqplib types will be accessible after installing just nestjs-rmq

Reply:
Adding it to devDependencies still allow you to acces those types. The only difference is that now you are sending them to production instead of development. I might be wrong, but I will run a test on my side to confirm.

Reply:
Yes, I am able, but if someone install our nestjs-rmq lib to microservice, he will not.

Reply:
Ok. I will remove this from the pr.

Reply:
Done.